docs/#596 remove redundant info from Data Archiving page (#814)
redpanda-data/documentation: docs/platform/data-management/data-archiving.mdx

This feature requires an [Enterprise license](../../introduction/licenses). To upgrade, contact [Redpanda sales](https://redpanda.com/try-redpanda?section=enterprise-cloud).
:::

When producers send events to Redpanda, Redpanda stores those events on a local storage volume.
When producers send events to Redpanda, Redpanda stores them on local storage.
Redpanda then replicates the data to other nodes,
according to the requirements defined by the producer.
Redpanda's persistence and replication processes protect your data in the event of a node or cluster failure. 
After configuring data archiving, your data is uploaded to cloud storage. 
In the event of a data center failure, data corruption, or cluster migration, you can recover your archived data from the cloud back to your cluster.

With data archiving, you enable remote write for topic backup to cloud storage, making data available for read operations only when it's in local storage. In the event of a data center failure, data corruption, or cluster migration, you can recover your archived data from the cloud back to your cluster.

:::note
Redpanda supports data archiving to Amazon S3 and Google Cloud Storage. 
:::

> ## Configure data archiving

Reviewer: So we have removed the data archiving config, but I now find this more confusing--data archiving is an enterprise license feature that we will provide no configuration guidance on, and we don't suggest that users rely on tiered storage instead. Can you help me understand how this improves the customer experience with this doc @mattschumpert ?

Reviewer 2: Data archiving is rather a user case that utilizes Tiered Storage than a feature. Even though, I'd agree to provide configuration guidance on it. Given this, the contents would look like:

* Introduction of the data archiving use case
* Typical configuration guidance; 
  * enable remote write (either cluster or topic level) and extend `retention.ms` and `retention.bytes` to unlimited to prevent the backup data from purging. 
  * enable remote read too to make it readable from cloud storage
* Required actions on a catastrophic event in the local cluster 
  * navigate to the Remote Recovery feature to recover the topic from archived data

Thoughts? I with the `local-target` props could be a hint on how much data we would guarantee to store locally, but it doesn't seem like a good idea to mention it according to Evgeny's comment on https://docs.google.com/document/d/13kVpsHeQM7xCgh9CG9fOgcnMHjtUn-Nvb2RzfxVWq6Y/edit?disco=AAAAj9Yqag0.

Author: Thank you, @daisukebe! I made Data Archiving a nested page under Tiered Storage and I added basic configuration with a link back to the Tiered Storage page for details. Please let me know what you think about this structure, and please add any corrections/additions to the content.
@mattschumpert: Please confirm if this is the correct messaging/positioning for this use case. Please add anything else you think would be helpful. 
Also: should the last section "Cancel and delete archiving jobs" be moved to the Tiered Storage page?

Author: @daisukebe, @mattschumpert: Is this [updated Data Archiving page](https://deploy-preview-814--redpanda-documentation.netlify.app/docs/platform/data-management/data-archiving/) (content and position under Tiered Storage) better than the [existing page](https://docs.redpanda.com/docs/platform/data-management/data-archiving/)? If so, I'd like to merge it by Friday. Please let me know any changed you'd like to see!

Reviewer 2: It looks basically neat to me, but it's a product decision. @mattschumpert, what do you think?

One concern to me is whether we should leave the `Cancel and delete archiving jobs` section as is since it looks obsolete or not user friendly. Confirming this with Evgeny over https://redpandadata.slack.com/archives/C02BDN76HUK/p1668671059249269. Can you postpone the merge to next week, if possible?

Author: > @Feediver1 Re: "is data archiving is an enterprise license feature..." I think this statement got copied over from the Tiered Storage page by mistake. TS requires an enterprise license, but data archiving is just a use case for TS, not a feature itself. @mattschumpert Please confirm that I got that right!
> 
> @micheleRP Can you remove that statement?

@AnneLeeperRP: I don't think we should remove the admonition. They need an enterprise license to use Data Archiving.

Reviewer 3: @micheleRP What if you say "data archiving uses Tiered Storage, which requires an enterprise license"?

This feature requires an [Enterprise license](../../introduction/licenses). To upgrade, contact [Redpanda sales](https://redpanda.com/try-redpanda?section=enterprise-cloud).
:::

> Data archiving is one Tiered Storage use case. With data archiving, you can enable remote write for simple topic backup to cloud storage. In the event of a data center failure, data corruption, or cluster migration, you can recover your archived data from the cloud back to your cluster. 

Reviewer: ```suggestion
This data archiving topic represents a single Tiered Storage use case. With data archiving, you can enable remote write for simple topic backup to cloud storage. In the event of a data center failure, data corruption, or cluster migration, you can recover your archived data from the cloud back to your cluster. 
```
